src/operator/dropout-inl.h

```diff
@@ -35,13 +35,14 @@ enum DropoutOpForwardResource {kRandom};
 namespace mxnet {
 namespace op {
 
-#if defined(USE_STATIC_MKL) && defined(_OPENMP)
+#if defined(USE_MKL) && defined(_OPENMP)
+const int nthr = omp_get_max_threads();
+std::vector<unsigned int> bernoulli_seed(nthr, 1);
```

Comment:
@piiswrong @mli The only reason why lint detect rand() function as a defect is that it is not thread-safe (non deterministic in multi threaded programs). So, all the other random functions which pass the lint check have seed values to track the sequences. If it does not need to be deterministic, could I exclude this line in the lint check (NOLINT) and use the basic rand() function? This rand() function is just used for generating seed numbers for mkl random number generators.